lib/converters/v1.0.0/converter-v1-to-v2.js

```diff
@@ -36,15 +36,17 @@ var _ = require('lodash').noConflict(),
             param;
 
         for (i = 0, ii = data.length; i < ii; i++) {
-            // clone param to avoid mutating data.
-            param = typeof data[i] === 'object' && Object.assign({}, data[i]);
-
-            // skip if param is missing property `key`,
-            // `key` is a required property, value can be null/undefined.
-            if (!(param && param.key)) {
+            // skip if:
+            //  1. param is not an object, or
+            //  2. param is missing property `key`
+            // @note `key` is a required property, value can be null/undefined.
+            if (typeof data[i] !== 'object' || _.isNil(data[i] && data[i].key)) {
```

Comment:
Under what cases would `data[i]` not be an object?

Reply:
Ideally, always an object.